packages/components/dropdown/src/dropdown.vue

```diff
@@ -126,11 +138,15 @@ export default defineComponent({
     const scrollbar = ref(null)
     const currentTabId = ref<string | null>(null)
     const isUsingKeyboard = ref(false)
+    const triggerKeys = [EVENT_CODE.enter, EVENT_CODE.space, EVENT_CODE.down]
 
     const wrapStyle = computed<CSSProperties>(() => ({
       maxHeight: addUnit(props.maxHeight),
     }))
     const dropdownTriggerKls = computed(() => [ns.m(dropdownSize.value)])
+    const triggerId = computed<string>(() => {
+      return props.id || useId().value
```

Comment:
**Suggestion:** we don't invoke computed inside a computed, which will always return a new computed.

```suggestion
const _id = useId()
  const triggerId = computed(() => {
      return props.id || _id.value
      })
```

Reply:
From what I remember it only recomputes if that inner referenced computed changes due to its refs updating, but there's no need to risk that in this case, yeah.

Reply:
Right, just in case